### PR TITLE
Add TiDB roadmap

### PR DESCRIPTION
**Pull request by queenypingcap**

### CODE REVIEW

**Comment by queenypingcap** [comment]
@shenli @ngaut

File: ROADMAP.md [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+- [ ] Executor
+  - [ ] Parallel Operators
+  - [ ] Compact Row Foramt to reduce memory usage
+  - [ ] File Sort
+- [ ] Support View
+- [ ] Support Window Function
+- [ ] Common Table Expression
+- [ ] Table Partition
+- [ ] Hash time index to reslove the issue with hot regions
+- [ ] Reversted Index
```
Comment (shenli) [inline ROADMAP.md]: Reverse Index

File: ROADMAP.md [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+  - [ ] File Sort
+- [ ] Support View
+- [ ] Support Window Function
+- [ ] Common Table Expression
+- [ ] Table Partition
+- [ ] Hash time index to reslove the issue with hot regions
+- [ ] Reverse Index
+- [ ] Cluster Index
+- [ ] Improve DDL
+- [ ] Foreign Key in the Restrict mode
```
Comment (ngaut) [inline ROADMAP.md]: I remembered that we have no plan to support foreign key.

**Comment by shenli** [comment]
LGTM

File: ROADMAP.md [hunk 3]
```diff
@@ -0,0 +1,69 @@
...
+- [ ] Tool
+  - [ ] Import distributed data
+  - [ ] Export distributed data
+  - [ ] Disaster Recovery
+- [ ] Flow control and degradation
+
+##  PD:
+- [ ] Improve namespace 
+  - [ ] Different replication policies for different namespaces and tables
+  - [ ] Dicentralize scheduling table regions
```
Comment (Darren) [inline ROADMAP.md]: Decentralize ?

Comment (queenypingcap) [reply ROADMAP.md]: Yes. Sorry about this.

**Pull request merged**